dist/src/main/resources/application.properties

```diff
@@ -6,9 +6,9 @@ management.endpoints.web.exposure.include=health,prometheus,loggers
 management.endpoint.prometheus.enabled=true
 management.metrics.export.prometheus.enabled=true
 management.endpoint.health.group.startup.include=gatewayStarted
-management.endpoint.health.group.startup.show-details=never
+management.endpoint.health.group.startup.show-details=always
```

Comment:
Just a quick check, but do we need this?

Reply:
No, that was for debugging, I'll remove it as I squash the commits before merging :)